MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/MessageService.java

```diff
@@ -34,9 +34,9 @@ public MessageService(final SimpMessagingTemplate messaging, final MessageSource
      * @param args - an array of arguments that will be filled in
      *             for params within the message, or null if none.
      */
-    public void broadcast(final String code, final String locale, final int user, final Object... args) {
-        String notNullLocale = locale == null ? "en" : locale;
-        final String msg = messageSource.getMessage(code, args, new Locale(notNullLocale));
+    public void broadcast(final String code, String locale, final int user, final Object... args) {
+        locale = locale == null ? "en" : locale;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "locale". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)